shared/src/ledger/storage/mod.rs

```diff
@@ -45,8 +46,8 @@ where
     pub header: Option<Header>,
     /// The height of the committed block
     pub last_height: BlockHeight,
-    /// The epoch of the current block
-    pub current_epoch: Epoch,
+    /// The epoch of the committed block
+    pub last_epoch: Epoch,
```

Comment:
Tte name wasn't correct - this is in fact the last committed block's epoch. The current epoch is inside `block.epoch` that is being updated on `BeginBlock`.